lib/cloudshaper/stack_module.rb

```
@@ -90,9 +90,15 @@ def register_resource(resource_type, name, &block)
     end
 
     def register_variable(name, &block)
+      return if @stack_elements[:variable].key?(name)
```

Should we silently ignore, or blow up hard?